.github/scripts/get-changed-modules-pull-request-test.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# Where is this script executing from ?
+BASEDIR=$(dirname "$0");pushd $BASEDIR 2>&1 >> /dev/null ;BASEDIR=$(pwd);popd 2>&1 >> /dev/null
+# echo "Running from directory ${BASEDIR}"
+export ORIGINAL_DIR=$(pwd)
+cd "${BASEDIR}"
+
+mkdir -p $BASEDIR/temp
+rm -fr $BASEDIR/temp/*output.txt
+
+
+if [[ "$(uname -o)" == "Darwin" ]]; then
+    echo "You are trying to run this test script on Mac, which doesn't support associative arrays or the declare -A syntax. Switch to linux, perhaps inside a docker container to test more"
```

Comment:
It would be a good idea to perhaps use the `docker run` command you used to run the `ubuntu:latest` container here, as I wouldn't know it off the top of my head

Reply:
Added.